packages/frontend/src/components/Chores/ChoresLayout.js

```diff
@@ -40,41 +51,79 @@ const ChoresLayout = () => {
     }));
   };
 
-  const handleAddChore = () => {
-    const chore = new Chore(
-      newChore.description,
-      newChore.deadline,
-      newChore.points,
-      newChore.assignee
-    );
+//  const handleAddChore = () => {
+//    const chore = new Chore(
+//      newChore.description,
+//      newChore.deadline,
+//      newChore.points,
+//      newChore.assignee
+//    );
+//
+//    setMyHouseholdChores((prevChores) => [
+//      ...prevChores,
+//      chore
+//    ]);
+//
+//    //will have to remove/change this once chores page is completed
+//    // Filter out chores that match the profile name and add them to My Chores
+//    if (
+//      chore.assignee.toLowerCase() ===
+//      "Johnny Clean".toLowerCase()
+//    ) {
+//      setMyChores((prevChores) => [...prevChores, chore]);
+//    }
+//
+//    setNewChore({
+//      description: "",
+//      deadline: new Date(),
+//      points: 0,
+//      assignee: ""
+//    });
+//  };
 
-    setMyHouseholdChores((prevChores) => [
-      ...prevChores,
-      chore
-    ]);
+  const addChore = async () => {
+    try {
+      const deadlineTimestamp = newChore.deadline.getTime();
+      const userIdObject = new mongoose.Types.ObjectId(userId);
```

Comment:
Were the mongoose types added to pass CORS checks?

Reply:
Yup, moving it to the backend though to make the deployment successful